sdk/python/feast/permissions/server/grpc.py

```diff
@@ -31,7 +32,7 @@ def grpc_interceptors(
     if auth_type == AuthManagerType.NONE:
         return None
 
-    return [AuthInterceptor()]
+    return [AuthInterceptor(), ErrorInterceptor()]
```

Comment:
Will it handle the exception happens in the `AuthInterceptor`?

Reply:
It won't. Specific code is needed in the `AuthInterceptor` (and maybe also use a Feast-specific error instead)

Reply:
I think that `ErrorInterceptor` should be called `if auth_type == AuthManagerType.NONE` as well

Reply:
@tmihalac yes, and also we need to move this interceptor in a different package, as it has nothing to do with the `feast.permission` package.
we'll review this later

Reply:
> Will it handle the exception happens in the AuthInterceptor?

@lokeshrangineni due to the latest refactory, we need to implement our own FeastAuthenticationError, so I would create a separate ticket for that.

Reply:
@lokeshrangineni created https://github.com/feast-dev/feast/issues/4466